wallet/src/de/schildbach/wallet/ui/main/MainViewModel.kt

```diff
@@ -263,7 +263,7 @@ class MainViewModel @Inject constructor(
     }
 
     private fun updateSyncStatus(state: BlockchainState) {
-        if (_isBlockchainSyncFailed.value != state.isSynced()) {
+        if (_isBlockchainSynced.value != state.isSynced()) {
```

Comment:
This is a bug that was preventing the "syncing" label from showing up unless the activity is relaunched.